msdb/msdb/templates/base.html

```diff
@@ -134,7 +134,7 @@
     </script>
     {% endcomment %}
     {% endblock inline_javascript %}
-
-    {% include "footer.html" %}
   </body>
+
+  
```

Comment:
generally very strange distribution of empty lines, but I would definitely not leave 2 empty lines